<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H5ConnectSettingsForm</class>
 <widget class="QDialog" name="H5ConnectSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="labelWebsocketPort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Web Socket Port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHttpPort">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>HTTP Port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinWebsocketPort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65534</number>
   </property>
   <property name="value">
    <number>50000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinHttpPort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65534</number>
   </property>
   <property name="value">
    <number>58000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinMessagingInterval">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelMessagingInterval">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Messaging Interval (ms):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>180</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
